README.md

```diff
@@ -14,6 +14,7 @@ The diagram below illustrates the general overview of what this SDK provides and
 flowchart TB
 agcdn[/Pantheon Advanced Global CDN\]-->ei[Pantheon Edge Integrations global library]
 ei-. optional .->smart_content_cdn(Smart Content CDN)
+smart_content_cdn-. optional .->smart_content_preview(Smart Content Preview)
```

Comment:
I think we need to add an optional step here to integrate the Smart Content ecosystem. Not least because `smart_content_preview` does _not_ require `smart_content_cdn`... I think the whole flowchart becomes:

```mermaid
flowchart TB
agcdn[/Pantheon Advanced Global CDN\]-->ei[Pantheon Edge Integrations global library]
ei-. optional .->smart_content(Smart Content)
smart_content-. optional .->smart_content_cdn(Smart Content CDN)
smart_content-. optional .->smart_content_preview(Smart Content Preview)
```